x/interchainstaking/keeper/msg_server.go

```diff
@@ -89,13 +89,54 @@ func (k msgServer) RequestRedemption(goCtx context.Context, msg *types.MsgReques
 			sdk.NewAttribute(types.AttributeKeyBurnAmount, msg.Value.String()),
 			sdk.NewAttribute(types.AttributeKeyRecipientAddress, msg.DestinationAddress),
 			sdk.NewAttribute(types.AttributeKeyChainID, zone.ChainId),
-			sdk.NewAttribute(types.AttributeKeyConnectionID, zone.ConnectionId),
```

Comment:
this was intentional?

Reply:
yes; connectionID is not a relevant field for MsgRequestRedemption.